config/rhoai/remove_admission_webhooks.yaml

```diff
@@ -0,0 +1,2 @@
+- op: remove
+  path: /webhooks
```

Comment:
Doesn't that remove all the webhooks? We only want to remove the `vpod.kb.io` and `mpod.kb.io` ones.

Reply:
You are correct, thanks Antonin. I have pushed another the relevant changes to target only vpod.kb.io and mpod.kb.io. Thank you :)